From 7a48d7ed38c693da6294b22f28c4a1d158dfb03b Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Sun, 25 Aug 2013 15:30:58 +0000
Subject: CVE-2013-1437 documentation fix

Module::Metadata executes code when gathering metadata about a
module by design. In versions previous to 1.000015 the
documentation stated, however, that Module::Metadata provides a
standard way to gather metadata about a .pm file without executing
unsafe code.

Origin: http://perl5.git.perl.org/perl.git/commit/68cdd4b5a461cacf43edfbc09b5490356b4a0fd0
Patch-Name: fixes/module_metadata_security_doc.diff
---
 cpan/Module-Metadata/lib/Module/Metadata.pm | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/cpan/Module-Metadata/lib/Module/Metadata.pm b/cpan/Module-Metadata/lib/Module/Metadata.pm
index e3c2504..4e416a8 100644
--- a/cpan/Module-Metadata/lib/Module/Metadata.pm
+++ b/cpan/Module-Metadata/lib/Module/Metadata.pm
@@ -764,8 +764,10 @@ Module::Metadata - Gather package and POD information from perl module files
 
 =head1 DESCRIPTION
 
-This module provides a standard way to gather metadata about a .pm file
-without executing unsafe code.
+This module provides a standard way to gather metadata about a .pm file through
+(mostly) static analysis and (some) code execution.  When determining the
+version of a module, the C<$VERSION> assignment is C<eval>ed, as is traditional
+in the CPAN toolchain.
 
 =head1 USAGE
 
